src/analysis/schwartz_space.lean

```diff
@@ -575,6 +577,162 @@ def mk_clm [ring_hom_isometric σ] (A : (D → E) → (F → G))
 
 end clm
 
+section multiplication
+
+variables [normed_add_comm_group D] [normed_space ℝ D]
+variables [normed_add_comm_group G] [normed_space ℝ G]
+
```

Comment:
can you introduce a definition for functions with tempered growth?

Reply:
You want a full-blown API with all the bells&whistles, i.e., coercions to functions, algebraic structure, inclusion mappings from Schwartz, etc?

Reply:
Definitely not for this PR, no! Just a predicate `has_tempered_growth`, to make things more readable in the statements.